IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/SSNewsCollectionViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SSNewsCollectionViewCell.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+import SDWebImage
+
+final class SSNewsCollectionViewCell: UICollectionViewCell {
+
+    static let identifier = "SSNewsCollectionViewCell"
+    static let Height = 80.0
+    
+    @IBOutlet weak var imgDetail: UIImageView!
+    @IBOutlet weak var lblDate: UILabel!
+    @IBOutlet weak var lblDescription: UILabel!
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblAuthor: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.contentView.frame = CGRect(x: self.contentView.frame.origin.x, y: self.contentView.frame.origin.y, width:UIScreen.main.bounds.width , height: CGFloat(SSNewsCollectionViewCell.Height))
```

Comment:
Is this required? Shouldn't be solved by the `getHeightForRow` (or something like that) method?

And overriding the width with `UIScreen.main.bounds.width` shouldn't be required.